localization/TimeSync.py

```diff
@@ -0,0 +1,257 @@
+"""Initial framework of temporal synchronisation of accelerometer, 
+gyroscope, gps-module and rgb camera (ongoing).
+
+Need logically debugged.
+
+Updated date: 10/04/23 by ychen441
+"""
+
+
+import io
+import sys
+import cv2
+import time
+import serial
+import pynmea2
+from tryCam import rs2stream
+from classOfuncs import helpers
+
+# Buffers for unsynchronised spatial measurements
+gps_buffer = []
+acc_buffer = []
+gyro_buffer = []
+# Buffers for temporal measurements
+t_std = []
+t_unsync_acc = []
+t_unsync_gyro = []
+# Buffers for synchronised data
+sync_acc = []
+sync_gyro = []
+
+# GPS data (lon, lat, time) fetching
+geo = helpers()
+ser = serial.Serial('/dev/ttyS0', 9600, timeout=5.0)
```

Comment:
Is 9600 satisfy our needs? Should we test it with sprayer?

Reply:
The port name and baud rate here are just examples for usage, we can use Win instead of Ubuntu and also other choices of baud rate to replace 9600.